src/rgw/rgw_sal_motr.cc

```diff
@@ -1444,6 +1444,22 @@ int MotrObject::set_obj_attrs(const DoutPrefixProvider* dpp, RGWObjectCtx* rctx,
       ldpp_dout(dpp, 0) <<__func__<< ": Failed to get key or object's entry from bucket index. rc= " << rc << dendl;
       return rc;
     }
+  // set attributes present in setattrs
```

Comment:
Given setattrs & attrs are same references, will this code (L1448-1452) would make any difference?

Reply:
attrs address is passed as setattrs ptr in majority of the cases i have seen till now. anyways change doesnt hurt